### DISCUSSION TITLE
feat: offline app

### DISCUSSION THREAD

**Discussion by Rayberk** [acikkaynak/deprem-yardim-frontend#549] [Ideas]
## Describe your request
The site should be able to operate offline as well.
If the site has previously been loaded, saving the data to local storage could help.
I think it would be better if, at least when loading, the previously collected data is displayed or if the people on the field can see it even when their Wi-Fi is gone.

## Describe the alternatives you evaluated
Saving data to local storage or caching locally.

Discord: Rayberk#8996

## Additional Context
Please provide any additional context or screen shots regarding this feature request.

**Comment by fdemir** [comment]
Is there a proposed solution?

**Comment by Rayberk** [comment]
Local storage has a limit of 5 MB, but there is no storage limit for indexeddb, which can be used instead.

**Comment by muratyilmaz743** [comment]
Here, caching may be more appropriate.

**Comment by Rayberk** [comment]
### How logical is it to store the latest data taken from the cache?
If the data taken is only a small area, it might not be useful to store it. That's why it might be more logical to merge all previously obtained data or give instructions to fetch all the data when the loading queue of the application has finished.

### How often is the backend data updated?
Isn't the frequency of updating the backend data already low? Storing it in local storage seems logical to me. It might be inconvenient to wait for it to be fetched again every time you enter if you are experiencing internet interruptions.
